### DISCUSSION TITLE
Exporting theme configuration to prevent having to reset each update

### DISCUSSION THREAD

**Discussion by BigBallard** [AtomMaterialUI/material-theme-issues#509] [Ideas]
Currently, every time material theme gets an update, I have to reset up my MT UI settings and its starting to get really arduous, especially when considering that I use like 5 different IDE's on three devices. QOL would significantly increase if we could somehow export the material UI settings we have setup and could simply import when an update happens and the settings need reconfigured.

**Comment by mallowigi** [comment]
Can you explain what you mean by "having to reset up your MT UI settings"? There should have been the case only when migrating from 8.X to 9.X
